source/organization/code-of-conduct.html.md

```diff
@@ -42,7 +42,7 @@ This Code of Conduct applies both within [project spaces](/organization/where) a
 
 ## Enforcement
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at ana@gelez.xyz. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at moderation@joinplu.me. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```

Comment:
as mentioned elsewhere, you should also mention who is on the moderation list, and how to contact these people individually, in case someone has a particular bad case with one of the moderators.

Reply:
Would adding a field with this information on the contributors page do the job?

Reply:
If this is put at another place, this other place must at least be mentioned/linked from here